src/applications/check-in/pre-check-in/pages/Error/index.jsx

```diff
@@ -21,36 +19,141 @@ import {
 import { useSessionStorage } from '../../../hooks/useSessionStorage';
 import Wrapper from '../../../components/layout/Wrapper';
 
+const appointmentAccordion = appointments => {
```

Comment:
Is this new for this ticket or was this moved from the old components and I just missed it?